app/user.go

```diff
@@ -583,11 +583,11 @@ func (a *App) GetUsersWithoutTeamPage(page int, perPage int, asAdmin bool, viewR
 }
 
 func (a *App) GetUsersWithoutTeam(offset int, limit int, viewRestrictions *model.ViewUsersRestrictions) ([]*model.User, *model.AppError) {
-	result := <-a.Srv.Store.User().GetProfilesWithoutTeam(offset, limit, viewRestrictions)
```

Comment:
You can simplify this with just `return a.Srv.Store.User().GetProfilesWithoutTeam(offset, limit, viewRestrictions)`